sktime/classification/base.py

```diff
@@ -61,7 +61,6 @@ class BaseClassifier(BaseEstimator, ABC):
         "capability:missing_values": False,
         "capability:train_estimate": False,
         "capability:contractable": False,
-        "capability:early_prediction": False,
```

Comment:
I guess this is safe without deprecation, as no classifier was using it? Is that correct?

Reply:
Yes should be safe, deprecate the tag itself just in case I think.

Reply:
ok, makes sense. If it was not used for anything so far.